### ISSUE TITLE
Adding Rachael Penfil to DSVA Slack

### ISSUE THREAD

**Issue by HeatherRienks** [department-of-veterans-affairs/va.gov-team#68637]
# Description

Please add Rachael Penfil, new Ad Hoc Accessibility Specialist on CHAMPVA team to DSVA Slack as a Full Member.
rachael.penfil@adhocteam.us

She should be added to #vfs-all-teams, #vfs-change-announcements and any other pertinent UX/Design and Accessibility  and VFS general channels for new team members.

Thanks!
